tools/coq_makefile.ml

```diff
@@ -412,6 +413,7 @@ let _ =
 
   let conf_file = Option.default "CoqMakefile" project.makefile ^ ".conf" in
   let local_file = Option.default "CoqMakefile" project.makefile ^ ".local" in
+  let dep_file = "." ^ Option.default "CoqMakefile" project.makefile in
```

Comment:
could you name it `.....coqdeps`? It makes it clear what the file is for, and makes .gitignore fixes more clear

Reply:
Yes, I did so, and changed @DEP_FILE@ to @COQDEPS@.